src/main/java/com/iexec/worker/feign/client/SmsClient.java

```diff
@@ -32,11 +33,16 @@
 public interface SmsClient {
 
     @PostMapping("/untee/secrets")
-    ResponseEntity<SmsSecretResponse> getUnTeeSecrets(@RequestHeader("Authorization") String authorization,
-                                                      @RequestBody WorkerpoolAuthorization workerpoolAuthorization) throws FeignException;
+    ResponseEntity<SmsSecretResponse> getUnTeeSecrets(
```

Comment:
I think we can remove /untee/*

Reply:
Yes, we will need a lot of cleaning for that part. But not in this PR IMHO.